### ISSUE TITLE
Ability to dedupe Pocket archive

### ISSUE THREAD

**Issue by lissertations** [hughrun/pickpocket#4]
Pickpocket is great, but it only works on one's pocket list, not archive. It would be useful to be able to dedupe an archive as well (particularly when used in conjunction with [pocket-snack](https://github.com/hughrun/pocketsnack))

**Comment by hughrun** [comment]
Good idea. This should be a simple matter of adding a flag to indicate whether to dedupe the List or the Archive, but I'll probably have to refactor the whole thing a bit to do it properly - which isn't necessarily a bad thing.

**Comment by hughrun** [comment]
Thinking about this more @lissertations it may make more sense to move this whole script into ‘pocketsnack’ as a pair of options (‘—dedupe —archive’ or ‘—dedupe —list)

**Comment by lissertations** [comment]
At the risk of creating more work for you, that might be a good way to go. Does this mean the deduping could only happen in conjunction with 'snacking'? Or could this process run independently?

**Comment by hughrun** [comment]
It would run independently but using the same program. I'll have to think about this but maybe I'll split the functions out into a proper module that could be imported independently (i.e. from another script).
